src/Landing/reduxx/actions/LatestArticles.jsx

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+import { FETCH_LATEST_POSTS } from '../types';
+
+const fetchLatestArticles = () => (dispatch) => {
+  axios.get('https://ah-backend-realers-staging.herokuapp.com/api/articles/?format=json&limit=9')
+    .then(res => res.data.results)
+    .then(articles => dispatch({
+      type: FETCH_LATEST_POSTS,
+      payload: articles,
+    }));
+};
+
+export default fetchLatestArticles;
```

Comment:
There's no jsx in this file, change the extension

Reply:
this has been changed.